sig-usability/annual-report-2021.md

```diff
@@ -0,0 +1,53 @@
+# 2021 Annual Report: SIG Usability
+
+## Operational
+
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate? have a CONTRIBUTING.md file?
+Yes, updated to reflect new meeting time
+https://github.com/kubernetes-sigs/sig-usability 
+
+All subprojects correctly mapped and listed in sigs.yaml?
+N/A
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+- Videos are all here: https://www.youtube.com/watch?v=cujQhg6JcgI&list=PL69nYSiGNLP0eY-U8DVJWHBwKvMDEtOxx 
+- We have small, interactive meetings, usually discuss and coordinate around our user research study that’s on-going 
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+N/A
+
+Same question as above but for working groups.
+N/A
+
+When was your last monthly community-wide update? (provide link to deck and/or recording)
+I do not recall 
+
+## Membership
+
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+We need to finalize the PR on kubernetes/kubernetes to make Gaby formally a chair 
+
+How do you measure membership? By mailing list members, OWNERs, or something else?
+We engage with membership via Slack and mailing list 
+
+How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+We are not focused on code PRs, we’re focused on staffing our user research study, which happens in Zoom. We’ve had steady growth of a small number of very senior user searchers from several different companies, and our goal is to do 2-4 in depth user research engagements a month.
+
+Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+We’ve documented our user research procedures and invite people to sign up to do shadowing and help take notes. As they participate they are invited to graduate to leading interviews.
```

Comment:
can you share those documents here?

Reply:
No, they are not open to the public. We take safe guarding the time and privacy of our research participants very seriously, and opening up our entire process could easily cause a troll problem.